src/qtiCommonRenderer/renderers/Math.js

```diff
@@ -39,15 +39,15 @@ export default {
     template: tpl,
     getContainer: containerHelper.get,
     render: function render(math) {
-        return new Promise(function(resolve) {
+        return new Promise(function (resolve) {
             var $item = containerHelper.get(math).closest('.qti-item');
             if (typeof MathJax !== 'undefined' && MathJax) {
                 //MathJax needs to be exported globally to integrate with tools like TTS, it's weird...
                 if (!window.MathJax) {
                     window.MathJax = MathJax;
                 }
                 //defer execution fix some rendering issue in chrome
-                if ($item.length && !$item.data('mathInitialized')) {
+                if ($item.length) {
                     $item.data('mathInitialized', true);
```

Comment:
you no need `$item.data('mathInitialized', true)` after you removed condition `!$item.data('mathInitialized')`